pages/book/send.mdx

```diff
@@ -131,7 +131,7 @@ There, the second message won't actually be sent:
 
 * After finishing the [compute phase][compute], the remaining value $\mathrm{R}$ of the contract is computed.
 
-* During the outbound message processing and assuming that there was enough value provided in the inbound message, the first message leaves $\mathrm{R} - (0.042 + \mathrm{forward-fees})$ [nanoToncoins](/book/integers#nanotoncoin) on the balance.
+* During the outbound message processing and assuming that there was enough value provided in the inbound message, the first message leaves $\mathrm{R} - (0.042 + \mathrm{\text{forward\_fees}})$ [nanoToncoins](/book/integers#nanotoncoin) on the balance.
```

Comment:
```suggestion
* During the outbound message processing and assuming that there was enough value provided in the inbound message, the first message leaves $\mathrm{R} - (0.042 + \mathrm{forward\_fees})$ [nanoToncoins](/book/integers#nanotoncoin) on the balance.
```

I suppose the idea was to make it look more like a variable, which is a good suggestion. But `\text` there is redundant :)

Reply:
In many editors, using the `\` within `\mathrm` can lead to errors.
![CleanShot 2024-08-29 at 09 15 25@2x](https://github.com/user-attachments/assets/4cefb0e4-2bf2-4dbc-9f2f-894e3d905c7b)

Reply:
@Laisky What matters here is how it renders in Tact docs, that's all

Reply:
And furthermore the doc framework we use, Nextra, uses a different LaTeX rendering engine — https://katex.org. So any MathJax playgrounds may report errors, which are unrelated to our case!